data/json/requirements/toolsets.json

```diff
@@ -35,6 +35,13 @@
     "//": "Forging of steel items (per steel chunk), charcoal forge is already a substitute for forge",
     "tools": [ [ [ "forge", 20 ], [ "oxy_torch", 20 ] ] ]
   },
+  {
+    "id": "blacksmithing_standard",
+    "type": "requirement",
+    "//": "Includes forging resources as well as tools needed for most blacksmithing",
+    "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 3 } ],
+    "tools": [ [ [ "forge", 20 ], [ "oxy_torch", 20 ] ], [ "tongs", -1 ] ]
```

Comment:
Shouldn't it be 50/10 instead of 20/20?

Reply:
We can take a look at what the ratio should be, a big part of the reason for this is to lock down that ratio instead of it being specified in an ad-hoc way everywhere.